src/test/java/edu/oakland/test/database/DatabasePersistentStorageStub.java

```diff
@@ -0,0 +1,47 @@
+package edu.oakland.test.database;
+
+import edu.oakland.helper.admin.LocationDataPoint;
+import edu.oakland.helper.admin.TrackData;
+import edu.oakland.production.database.DatabasePersistentStorage;
+
+public class DatabasePersistentStorageStub implements DatabasePersistentStorage {
+  private int rfid;
+  private TrackData trackData;
+  
+  public DatabasePersistentStorageStub(int rfid, TrackData trackdata) {
+    this.rfid = rfid;
+    this.trackData = trackdata;
+  }
+  public int locateRfidData() {
+    return rfid;
+  }
+
+  public LocationDataPoint getLocationDataPoint(int offset) {
+    return null;
+  }
+
+  public void storeLocationDataPoint(LocationDataPoint locationDataPoint) {
+
+  }
+
+  public TrackData getTrackData(int offset) {
+    return trackData;
+  }
+
+  public void storeTrackData(TrackData trackData) {
+
```

Comment:
This should set up the setter:
```suggestion
    this.trackData = trackData;
```